assets/scss/base/_lists.scss

```diff
@@ -58,11 +48,81 @@ ul, nav ul {
 }
 
 .list--inline {
-	li {
-		display: inline;
+  li {
+    display: inline;
+  }
+}
+
+/*
+Numbered List
+
+Defining sections in markup with numbers associated against them using `.list--numbered` on the container and `.numbered` on the element to identify its position
+
+Markup:
+<fieldset class="list--numbered">
+  <legend>Section 1</legend>
+  <div class="form-field-group numbered">
```

Comment:
+ there is only one control in these `divs` so `form-field` feels "more" correct